include/circt/Dialect/SV/Statements.td

```diff
@@ -143,3 +143,43 @@ def FatalOp : SVOp<"fatal"> {
 
   let assemblyFormat = [{ attr-dict }];
 }
+
+//===----------------------------------------------------------------------===//
+// Verification Statements
+//===----------------------------------------------------------------------===//
+
+def AssertOp : SVOp<"assert"> {
+  let summary = "immediate assertion statement";
+  let description = [{
+    Assert that a specific condition is always true.
+  }];
+
+  let arguments = (ins AnyType:$predicate);
+  let results = (outs);
+
+  let assemblyFormat = [{ attr-dict `(` $predicate `)` `:` type($predicate) }];
```

Comment:
It's a bit subjective, but I'd drop the `(`'s from around the operand list in the asm syntax when it isn't "call like".  Similarly in the assume/cover op as well.  I'm not sure how consistent we are here, but `sv.if` and others don't put parens around their arguments.

Reply:
Good style suggestion. Will fixup.

Reply:
Suggestion applied in a fixup commit.